Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'sqlite3', '~> 1.3.6'
 
 # Capybara versions that support RSpec 3 only support RUBY_VERSION >= 1.9.3
 if RUBY_VERSION >= '1.9.3'
-  gem 'capybara', '~> 2.2.0', :require => false
+  gem 'capybara', '~> 2.2', :require => false
```

Comment:
We understand that. It's not practical for us to slice down every version in our build matrix (we already slice about 30 ruby and rails combinations). Semantic versioning from capybara says they shouldn't break stuff going forward, so the testing impetus really falls to them here.